WordPress/Classes/ViewRelated/System/Notices/NoticePresenter.swift

```diff
@@ -61,13 +61,9 @@ class NoticePresenter: NSObject {
     private init(store: NoticeStore) {
         self.store = store
 
```

Comment:
I know it's pre-existing, but I'm not sure this needs to be exposed to Objective-C anymore, so maybe we can remove the inheritance from NSObject?

Reply:
This was a great catch! I was thinking the notification observers required this for some reason.

Thanks!